mobile/lib/infrastructure/repositories/local_album.repository.dart

```diff
@@ -226,21 +239,49 @@ class DriftLocalAlbumRepository extends DriftDatabaseRepository
     });
   }
 
-  Future<void> _addAssets(String albumId, Iterable<LocalAsset> assets) {
-    if (assets.isEmpty) {
+  @override
+  Future<List<LocalAsset>> getAssetsToHash(String albumId) {
```

Comment:
Can you please check this query, it is currently getting all the assets in the table instead of from the `albumId`?

Reply:
nvm, fixed it